lib/ansible/plugins/callback/logstash.py

```diff
@@ -86,16 +96,18 @@ def __init__(self):
         if not HAS_LOGSTASH:
             self.disabled = True
             self._display.warning("The required python-logstash is not installed. "
-                                  "pip install python-logstash")
+                                  "pip install python3-logstash")
```

Comment:
This should probably be conditional, based on whether using py2 or py3.  You can use `ansible.module_utils.six.PY3` for this logic.